ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,36 @@
-<!-- Check if the title is descriptive! -->
+<!-- Check if the title issue is descriptive! -->
 - Parent issues: -
 - Relevant issues: -
 - Estimated effort: -
 
 * * *
 
 ## User story
-The user story for the issue.
+<!-- The user story for the issue. Remove if not applicable -->
+As a...
 
 ## What
-This issue demands 
-<!-- Specify in more detail what this issue adds to the repository. -->
+<!-- Specify what this issue adds to the project -->
+This issue demands...
 
 ## Why
-This issue is needed for the project because 
-<!-- Specify in more detail why this issue is needed in the project. -->
+<!-- Specify why this issue is needed in the project -->
+This issue is needed for the project because...
 
 ## How
-This feature can be added by 
-<!-- Specify in general terms how this feature can be added. -->
+<!-- Specify how this feature can be added to the project -->
+This feature can be added by...
 
 ## Definition of ready
+<!-- Specify any issues that are still open that must be completed before people can start on this issue -->
 This features is not ready until the following issues are finished:
-
-- 
-<!-- Specify if there are still issues open that must be completed first before people can start on this issue -->
+- Placeholder
```

Comment:
Do we want this? We fill (almost) never something in here so we do have to remove Placeholder every time. It would be better if you removed Placeholder or replaced it by something like "Waiting on other issues is not needed"